asm/non_matchings/code_80071F00/func_80085EF8.s

```diff
@@ -22,12 +22,12 @@ glabel func_80085EF8
 /* 086B2C 80085F2C 44058000 */  mfc1  $a1, $f16
 /* 086B30 80085F30 0C022DF5 */  jal   func_8008B7D4
 /* 086B34 80085F34 00000000 */   nop   
-/* 086B38 80085F38 3C050600 */  lui   $a1, %hi(D_06006800) # $a1, 0x600
-/* 086B3C 80085F3C 3C060601 */  lui   $a2, %hi(D_0600A000) # $a2, 0x601
+/* 086B38 80085F38 3C050600 */  lui   $a1, %hi(d_course_rainbow_road_boo_tlut_list) # $a1, 0x600
+/* 086B3C 80085F3C 3C060601 */  lui   $a2, %hi(d_course_rainbow_road_neon_boo) # $a2, 0x601
```

Comment:
Consider keeping 'neon' for the tlut name. I just think people might get confused since the two go together.

Reply:
I though I had done that actually, good catch